packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -166,7 +166,7 @@ class SectionHeader extends React.Component {
 
     return (
       <Element {...headerAttributes} className={sectionHeaderClassNames} aria-label={!onClick ? headerText : undefined}>
-        <ArrangeWrapper {...buttonAttributes} className={cx('arrange-wrapper', { 'title-fixed': isTitleFixed })}>
+        <ArrangeWrapper {...buttonAttributes} className={cx('arrange-wrapper', { 'title-fixed': isTitleFixed, 'title-sticky': headerAttributes.isTitleSticky })}>
```

Comment:
Is there a reason why this is a custom prop instead of a regular prop?

Reply:
Currently there's only one usage for this through the table sections. But it can be changed to regular prop if required.